docs/copy-secrets-to-vault.md

```diff
@@ -0,0 +1,78 @@
+# `copy-secrets-to-vault`
+This `vault-radar` command is used for copying unmanaged secrets discovered in a supported scan into HashiCorp Vault.
+
+## Usage
+
+### Supported Scans
+
+We can copy secrets from scans that support this functionality. Those are currently:
+
+- `scan-folder`
+- `scan-aws-s3`
+
+If you'd like additional scan types to be added, please reach out and give us that feedback!
+
+### A quick intro/setting the scene
+
+`copy-secrets-to-vault` copies unmanaged secrets, discovered in a scan, to HashiCorp Vault. As a result, it needs to know which secrets are already managed.
+
+It figures this out by looking at the outfile of a previous supported scan that used a Vault index file. For example, if you ran
+
+```bash
+vault-radar scan-folder -o scan-folder-outfile --index-file vault-scan-outfile
+```
+
+where `vault-scan-outfile` is the result of a `scan-vault`, this command would accept as input `scan-folder-outfile` to understand which secrets need to be managed.
+
+`copy-secrets-to-vault` will then re-run the previous scan (characterized by `scan-folder-outfile` in the above example), gather the unmanaged secrets, and Vault them.
```

Comment:
nit: Should we explicitly say re-run the previous scan with same arguments? There is a note right after this line but we don't want users to think that this only applies to the limit flag right